src/Stripe.net/Services/StripeParamsBase.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace Stripe
+{
+    public class StripeParamsBase
```

Comment:
I like the new name but should it be `StripeBaseOptions` to match the rest of the class naming?

Reply:
Yeah, that's probably a better name.